posix.mak

```diff
@@ -39,10 +39,18 @@ REMOTE_DIR=d-programming@digitalmars.com:data
 TMP?=/tmp
 
 # Last released versions
+# By default the master branch is built under {library,phobos}-prerelease
+# with USE_MASTER_AS_STABLE=1 you can build the master branch instead of a stable release
```

Comment:
Actually one comment - I think all such hacks should be accompanied with a verbose explanation of why it's needed, and implied or explicit terms for when the hack can be removed.